src/textdescriptives/components/quality_data_classes.py

```diff
@@ -232,6 +243,7 @@ def passed(self) -> bool:
                 self.duplicate_paragraph_chr_fraction.passed,
                 all(v.passed for v in self.duplicate_ngram_chr_fraction.values()),
                 all(v.passed for v in self.top_ngram_chr_fraction.values()),
+                self.oov_ratio.passed,
```

Comment:
We should change this to allow None and then have self.oov_ratio.passed be None.